modules/integration/tests-ui/src/test/resources/testng-server-mgt.xml

```diff
@@ -24,6 +24,7 @@
             <!--<class name="org.wso2.carbon.am.integration.ui.APIMANAGER2957EpochTimeTestCase"/>
             <class name="org.wso2.carbon.am.integration.ui.APIMANAGER3017ExternalStoreLifeCycleTestCase"/>
             <class name="org.wso2.carbon.am.integration.ui.APIMANAGER3149SSOLogin"/>-->
+            <!--<class name="org.wso2.carbon.am.integration.ui.APIResourceBodyParameterRemovalTestCase"/>-->
```

Comment:
I disabled that test case because all the other UI test cases are disabled. Had a chat with a team member for the reason. He mentioned that builds getting failed if we enable the UI tests